docs_src/query_params_str_validations/tutorial006d_an_py310.py

```diff
@@ -0,0 +1,19 @@
+from typing import Annotated, Optional, Union
```

Comment:
```suggestion
from typing import Optional, Union
from typing_extensions import Annotated
```

Otherwise it fails for Python 3.8

Reply:
Thanks @YuriiMotov!
I’ve updated th import to use typing_extensions.Annotated for Python 3.8 compatibility ✅
Just pushed the fix , everything should be ready now!